src/translators/sql_translator.py

```diff
@@ -1122,10 +1123,14 @@ def query(self,
             try:
                 self.cursor.execute(op)
 
+            except crate.client.exceptions.ProgrammingError as e:
+                err_msg = self.sql_error_handler(e)
+                self.logger.error(str(e),exc_info=True)
+                entities = []
+                if err_msg:
+                    message = err_msg
```

Comment:
@NEC-Vishal not sure what this is for? If you could please explain, maybe you can share some tests? Also as a side note, we can't put this code in here b/c it's Crate specific, so if that's a fix, we've got to push it down to the Crate translator. Btw, any idea what's happening in Timescale land?

Thanks alot for taking on this PR, much appreciated!